src/sif/motorcyclecost.cc

```diff
@@ -242,8 +241,7 @@ class MotorcycleCost : public DynamicCost {
   virtual Cost TransitionCostReverse(const uint32_t idx,
                                      const baldr::NodeInfo* node,
                                      const baldr::DirectedEdge* pred,
-                                     const baldr::DirectedEdge* edge,
-                                     const bool has_traffic) const;
+                                     const baldr::DirectedEdge* edge) const override;
```

Comment:
You can remove comment for `has_traffic` from both function's doc comments.

Reply:
Good catch! 👍